raiden/network/transport/matrix/transport.py

```diff
@@ -623,7 +623,7 @@ def _user_id(self) -> Optional[str]:
     @property
     def network_id(self) -> ChainID:
```

Comment:
nitpick: But since you are touching all of these can you change network_id() to chain_id() here? I think that the terminology we want at the end is chain_id.

Reply:
Sure.